schemas/app/inventory/StockMovementItem.json

```diff
@@ -26,6 +26,13 @@
       "target": "Location",
       "create": true
     },
+    {
+      "fieldname": "batchNumber",
```

Comment:
Why is there just a single `batchNumber` field here.

Shouldn't there be `fromBatch` and `toBatch` fields, a user can choose to transfer stock out of a specific batch too, not just into.

Reply:
It's better to have multiple item rows in case of multiple batch. It will be easier for the user to quickly map qty and batch from the table rows and also is easier for devs to handle. Having `fromBatch` and `toBatch` won't give us how much qty to increase/decrease from a particular batch.

Reply:
I'm not disputing whether multiple-item rows should be present, the number of rows depends on the number of unique Batch and Location combinations on either side of a transfer for an Item.

---

Let me reiterate, if you're moving out 1 quantity of a batched Item from Point A. Point A needs to be well-defined.

Point A in this case is a Location, Batch tuple. 

Only in the case that there happens to be just a single Batch can you forgo mentioning the Batch.

Else, if you don't mention the Batch of Point A, which Batch are you going to deduct the quantity from?

---

That being said, I didn't understand this: 

>  Having fromBatch and toBatch won't give us an idea of how much qty to increase/decrease from a particular batch.

Why should this be an issue? If anything, having `fromBatch` and `toBatch` should be making it explicit as to which batch has to be decreased and which increased.

---

If I have misunderstood your implementation please make it clearer with an example.